docs/tutorials/NMR/Example_1_-Ethanol.md

```diff
@@ -0,0 +1,115 @@
+<img alt="Fig1. Proton spectrum of ethanol" src="../../img/nmr_tut1.png" width = "300"/>
+<figure fig1>
+  <figcaption>Fig1. Proton spectrum of ethanol</figcaption>
+</figure>
+
+
+The discovery that one could actually see chemical shifts in hydrogen spectra was made in 1951 at Stanford University by Packard, Arnold, Dharmatti (shown in Fig1. ). We will try to reproduce this result.  
+
+
+## Files   
+
+
+*ethanol.cell*
+```
```

Comment:
It should also be possible to set these properly as titles of the code fences
https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#adding-a-title

Reply:
How do you want it to look exactly? I used the same formatting as the other tutorials for the file titles (unless you meant something else)

Reply:
Given you are writing the docs, you have the power to style these however you wish. Personally, I find the titled code-fences style attaches the name nicely to the block making it clear, but that's a stylistic choice rather than content.